subsys/ll_sched/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig LL_SCHEDULER
+	bool "LL Scheduler"
+	default n
+	help
+	  This option enables the low latency (LL) scheduler. It is derived
+	  from the scheduler used in the audio SOF module. Currently, it only
```

Comment:
no need to have those references to SOF everywhere, just put this in the docs somewhere as reference and that should be enough

Reply:
Done.